spmd/tensor/device_mesh.py

```diff
@@ -211,44 +217,78 @@ def get_rank(self) -> int:
         return get_rank()
 
     def scatter(
```

Comment:
Maybe write a docstring to clarify semantic.

Reply:
added a docstr for `scatter`, will add more docstr for other collectives in following PRs.